src/main/java/me/totalfreedom/totalfreedommod/command/Command_masterbuilderworld.java

```diff
@@ -176,6 +174,7 @@ else if (args[0].equals("weather"))
     }
 
     // TODO: Redo this properly
+    // ^^^
```

Comment:
Why is this added?

Reply:
Good question, I don't remember

Reply:
fixed, I still don't know why I put that there